package.json

```diff
@@ -66,6 +67,10 @@
     "istanbul": "0.4.5",
     "mocha": "3.1.2",
     "npm-run-all": "3.1.1",
+    "postcss": "5.2.4",
+    "postcss-cli": "2.6.0",
+    "postcss-cssnext": "2.8.0",
+    "postcss-normalize": "0.3.0",
```

Comment:
I am definitely having issues with this: `postcss-normalize` is now deprecated, we should not rely on it.
It is suggesting to use `postcss-import`, but at that point we can use `@import` just fine as it's supported in all browsers we run on (cf. previous comments about `postcss-import`).

But more generally, as @williamboman is pointing out, we should stay with `normalize.css` v3.0.1 to avoid breakage. Bumping `normalize.css` regardless of Bootstrap is asking for trouble IMO, and is very unlikely to bring value.

I think that, as long as we are using Bootstrap, we should stick with the version of `normalize.css` that's in there. When/If we get rid of Bootstrap, then I'll be all in favor of reconsidering this.